lib/alchemy/upgrader/four_point_one.rb

```diff
@@ -16,7 +16,7 @@ def alchemy_4_1_todos
         Changed tagging provider to Gutentag
         ------------------------------------
 
-        The automatic updater that just ran updated all existing `acts_as_taggable_on_migrations`,
+        The automatic updater that just run updated all existing `acts_as_taggable_on_migrations`,
```

Comment:
I don't think we need to change that. This is the note that gets displayed after the upgrader ran.

Reply:
I think I must return to school for a better English...sorry!